src/main/java/org/skife/jdbi/v2/util/EnumColumnMapperFactory.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2015 Zane Benefits
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.skife.jdbi.v2.util;
+
+import org.skife.jdbi.v2.ResultColumnMapperFactory;
+import org.skife.jdbi.v2.StatementContext;
+import org.skife.jdbi.v2.tweak.ResultColumnMapper;
+
+public enum EnumColumnMapperFactory implements ResultColumnMapperFactory {
+    BY_NAME() {
```

Comment:
I could have sworn they were required (like for anonymous inners) but the compiler says they're ok. So.. yes.